src/Core/Domain/Feature/Exception/FeatureConstraintException.php

```diff
@@ -31,13 +31,7 @@
  */
 class FeatureConstraintException extends FeatureException
 {
-    /**
-     * Code is used when feature does not have name.
-     */
-    public const EMPTY_NAME = 1;
+    public const INVALID_NAME = 1;
 
-    /**
-     * Used when feature name is invalid.
-     */
-    public const INVALID_NAME = 2;
+    public const INVALID_POSITION = 2;
```

Comment:
this kind of things are BC breaks I think. But the new feature page wasn't yet open for use, so think we should still be able to do such things? (probably nobody used it yet?).